tests/unit_tests/db_engine_specs/test_doris.py

```diff
@@ -120,6 +121,11 @@ def test_adjust_engine_params(
     returned_url, returned_connect_args = DorisEngineSpec.adjust_engine_params(
         url, connect_args
     )
+    # # Update expected schema to include .information_schema
+    # if return_schema not in ".":
+    #     expected_schema = f"{return_schema}.information_schema"
+    # else:
+    #     expected_schema = return_schema
```

Comment:
Why is this commented out block included here?

Reply:
Sorry, this is redundant, I will delete it